jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/sockets/helper/KeepAliveHelper.kt

```diff
@@ -0,0 +1,34 @@
+package org.jellyfin.sdk.api.sockets.helper
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import mu.KotlinLogging
+import org.jellyfin.sdk.api.sockets.SocketInstance
+import org.jellyfin.sdk.model.socket.KeepAliveMessage
+import kotlin.time.Duration.Companion.seconds
+
+private val logger = KotlinLogging.logger {}
+
+internal class KeepAliveHelper(
+	private val coroutineScope: CoroutineScope,
+) {
+	private var keepAliveTicker: Job? = null
+
+	fun reset(instance: SocketInstance, lostTimeout: Int) {
```

Comment:
I'd suggest adding explicit function docs explaining the lostTimeout is in seconds (even if the class is internal).